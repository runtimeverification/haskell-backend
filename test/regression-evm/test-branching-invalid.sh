#!/bin/sh
${KORE_EXEC:?} test-branching-invalid-definition.kore --module ETHEREUM-SIMULATION --pattern test-branching-invalid-tmp.in.AscNlhsvEB --searchType FINAL --search test-branching-invalid-tmp.pattern.OvSVVYTtn5 "$@"
