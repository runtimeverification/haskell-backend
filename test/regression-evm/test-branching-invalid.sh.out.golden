/* Spa */
\and{SortGeneratedTopCell{}}(
    /* Spa */
    \and{SortGeneratedTopCell{}}(
        /* Spa */
        \and{SortGeneratedTopCell{}}(
            /* Spa */
            \and{SortGeneratedTopCell{}}(
                /* Spa */
                \and{SortGeneratedTopCell{}}(
                    /* Spa */
                    \and{SortGeneratedTopCell{}}(
                        /* Spa */
                        \and{SortGeneratedTopCell{}}(
                            /* Spa */
                            \and{SortGeneratedTopCell{}}(
                                /* Spa */
                                \and{SortGeneratedTopCell{}}(
                                    /* Spa */
                                    \and{SortGeneratedTopCell{}}(
                                        /* Spa */
                                        \and{SortGeneratedTopCell{}}(
                                            /* Spa */
                                            \and{SortGeneratedTopCell{}}(
                                                /* Spa */
                                                \and{SortGeneratedTopCell{}}(
                                                    /* Spa */
                                                    \and{SortGeneratedTopCell{}}(
                                                        /* Spa */
                                                        \and{SortGeneratedTopCell{}}(
                                                            /* Spa */
                                                            \and{SortGeneratedTopCell{}}(
                                                                /* Spa */
                                                                \equals{SortOutputCell{}, SortGeneratedTopCell{}}(
                                                                    /* Fl Fn D Sfa */
                                                                    Var'Unds'0:SortOutputCell{},
                                                                    /* Fl Fn D Sfa Cl */
                                                                    Lbl'-LT-'output'-GT-'{}(
                                                                        /* Fl Fn D Sfa Cl */
                                                                        \dv{SortBytes{}}(
                                                                            ""
                                                                        )
                                                                    )
                                                                ),
                                                                /* Spa */
                                                                \equals{SortEndPCCell{}, SortGeneratedTopCell{}}(
                                                                    /* Fl Fn D Sfa */
                                                                    Var'Unds'1:SortEndPCCell{},
                                                                    /* Fl Fn D Sfa Cl */
                                                                    Lbl'-LT-'endPC'-GT-'{}(
                                                                        /* Fl Fn D Sfa Cl */
                                                                        \dv{SortInt{}}("11")
                                                                    )
                                                                )
                                                            ),
                                                            /* Spa */
                                                            \equals{SortCallStackCell{}, SortGeneratedTopCell{}}(
                                                                /* Fl Fn D Sfa */
                                                                Var'Unds'2:SortCallStackCell{},
                                                                /* Fl Fn D Sfa Cl */
                                                                Lbl'-LT-'callStack'-GT-'{}(
                                                                    /* Fl Fn D Sfa Cl */
                                                                    /* InternalList: */ Lbl'Stop'List{}()
                                                                )
                                                            )
                                                        ),
                                                        /* Spa */
                                                        \equals{SortInterimStatesCell{}, SortGeneratedTopCell{}}(
                                                            /* Fl Fn D Sfa */
                                                            Var'Unds'3:SortInterimStatesCell{},
                                                            /* Fl Fn D Sfa Cl */
                                                            Lbl'-LT-'interimStates'-GT-'{}(
                                                                /* Fl Fn D Sfa Cl */
                                                                /* InternalList: */ Lbl'Stop'List{}()
                                                            )
                                                        )
                                                    ),
                                                    /* Spa */
                                                    \equals{SortTouchedAccountsCell{}, SortGeneratedTopCell{}}(
                                                        /* Fl Fn D Sfa */
                                                        Var'Unds'4:SortTouchedAccountsCell{},
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'touchedAccounts'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            /* InternalSet: */ Lbl'Stop'Set{}()
                                                        )
                                                    )
                                                ),
                                                /* Spa */
                                                \equals{SortCallStateCell{}, SortGeneratedTopCell{}}(
                                                    /* Fl Fn D Sfa */
                                                    Var'Unds'5:SortCallStateCell{},
                                                    /* Fl Fn D Sfa Cl */
                                                    Lbl'-LT-'callState'-GT-'{}(
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'program'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortBytes{}}(
                                                                "`\x03`\x04\x01`\x08\x14`\x0cW\xfe[\x00"
                                                            )
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'jumpDests'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            /* InternalSet: */ /* concrete element: */ LblSetItem{}(
                                                                /* Inj: */ inj{SortInt{}, SortKItem{}}(
                                                                    \dv{SortInt{}}("12")
                                                                )
                                                            )
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'id'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            Lbl'Stop'Account'Unds'EVM-TYPES'Unds'Account{}()
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'caller'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            Lbl'Stop'Account'Unds'EVM-TYPES'Unds'Account{}()
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'callData'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortBytes{}}("")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'callValue'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortInt{}}("0")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'wordStack'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            Lbl'Stop'WordStack'Unds'EVM-TYPES'Unds'WordStack{}()
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'localMem'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortBytes{}}("")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'pc'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortInt{}}("11")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'gas'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortInt{}}("9972")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'memoryUsed'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortInt{}}("0")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'callGas'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortInt{}}("0")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'static'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortBool{}}("false")
                                                        ),
                                                        /* Fl Fn D Sfa Cl */
                                                        Lbl'-LT-'callDepth'-GT-'{}(
                                                            /* Fl Fn D Sfa Cl */
                                                            \dv{SortInt{}}("0")
                                                        )
                                                    )
                                                )
                                            ),
                                            /* Spa */
                                            \equals{SortSubstateCell{}, SortGeneratedTopCell{}}(
                                                /* Fl Fn D Sfa */
                                                Var'Unds'6:SortSubstateCell{},
                                                /* Fl Fn D Sfa Cl */
                                                Lbl'-LT-'substate'-GT-'{}(
                                                    /* Fl Fn D Sfa Cl */
                                                    Lbl'-LT-'selfDestruct'-GT-'{}(
                                                        /* Fl Fn D Sfa Cl */
                                                        /* InternalSet: */ Lbl'Stop'Set{}()
                                                    ),
                                                    /* Fl Fn D Sfa Cl */
                                                    Lbl'-LT-'log'-GT-'{}(
                                                        /* Fl Fn D Sfa Cl */
                                                        /* InternalList: */ Lbl'Stop'List{}()
                                                    ),
                                                    /* Fl Fn D Sfa Cl */
                                                    Lbl'-LT-'refund'-GT-'{}(
                                                        /* Fl Fn D Sfa Cl */
                                                        \dv{SortInt{}}("0")
                                                    ),
                                                    /* Fl Fn D Sfa Cl */
                                                    Lbl'-LT-'accessedAccounts'-GT-'{}(
                                                        /* Fl Fn D Sfa Cl */
                                                        /* InternalSet: */ Lbl'Stop'Set{}()
                                                    ),
                                                    /* Fl Fn D Sfa Cl */
                                                    Lbl'-LT-'accessedStorage'-GT-'{}(
                                                        /* Fl Fn D Sfa Cl */
                                                        /* InternalMap: */ Lbl'Stop'Map{}()
                                                    )
                                                )
                                            )
                                        ),
                                        /* Spa */
                                        \equals{SortGasPriceCell{}, SortGeneratedTopCell{}}(
                                            /* Fl Fn D Sfa */
                                            Var'Unds'7:SortGasPriceCell{},
                                            /* Fl Fn D Sfa Cl */
                                            Lbl'-LT-'gasPrice'-GT-'{}(
                                                /* Fl Fn D Sfa Cl */
                                                \dv{SortInt{}}("0")
                                            )
                                        )
                                    ),
                                    /* Spa */
                                    \equals{SortOriginCell{}, SortGeneratedTopCell{}}(
                                        /* Fl Fn D Sfa */
                                        Var'Unds'8:SortOriginCell{},
                                        /* Fl Fn D Sfa Cl */
                                        Lbl'-LT-'origin'-GT-'{}(
                                            /* Fl Fn D Sfa Cl */
                                            Lbl'Stop'Account'Unds'EVM-TYPES'Unds'Account{}()
                                        )
                                    )
                                ),
                                /* Spa */
                                \equals{SortBlockhashesCell{}, SortGeneratedTopCell{}}(
                                    /* Fl Fn D Sfa */
                                    Var'Unds'9:SortBlockhashesCell{},
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'blockhashes'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */
                                        /* InternalList: */ Lbl'Stop'List{}()
                                    )
                                )
                            ),
                            /* Spa */
                            \equals{SortBlockCell{}, SortGeneratedTopCell{}}(
                                /* Fl Fn D Sfa */ Var'Unds'10:SortBlockCell{},
                                /* Fl Fn D Sfa Cl */
                                Lbl'-LT-'block'-GT-'{}(
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'previousHash'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'ommersHash'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'coinbase'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'stateRoot'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'transactionsRoot'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'receiptsRoot'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'logsBloom'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */
                                        \dv{SortBytes{}}("")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'difficulty'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'number'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'gasLimit'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'gasUsed'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'timestamp'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'extraData'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */
                                        \dv{SortBytes{}}("")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'mixHash'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'blockNonce'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'-LT-'ommerBlockHeaders'-GT-'{}(
                                        /* Fl Fn D Sfa Cl */
                                        LblJSONList{}(
                                            /* Fl Fn D Sfa Cl */
                                            Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}()
                                        )
                                    )
                                )
                            )
                        ),
                        /* Spa */
                        \equals{SortKCell{}, SortGeneratedTopCell{}}(
                            /* Fl Fn D Sfa */ Var'Unds'11:SortKCell{},
                            /* Fl Fn D Sfa Cl */
                            Lbl'-LT-'k'-GT-'{}(
                                /* Fl Fn D Sfa Cl */
                                kseq{}(
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'Hash'halt'Unds'EVM'Unds'KItem{}(),
                                    /* Fl Fn D Sfa Cl */ dotk{}()
                                )
                            )
                        )
                    ),
                    /* Spa */
                    \equals{SortExitCodeCell{}, SortGeneratedTopCell{}}(
                        /* Fl Fn D Sfa */ Var'Unds'12:SortExitCodeCell{},
                        /* Fl Fn D Sfa Cl */
                        Lbl'-LT-'exit-code'-GT-'{}(
                            /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("1")
                        )
                    )
                ),
                /* Spa */
                \equals{SortModeCell{}, SortGeneratedTopCell{}}(
                    /* Fl Fn D Sfa */ Var'Unds'13:SortModeCell{},
                    /* Fl Fn D Sfa Cl */
                    Lbl'-LT-'mode'-GT-'{}(/* Fl Fn D Sfa Cl */ LblNORMAL{}())
                )
            ),
            /* Spa */
            \equals{SortScheduleCell{}, SortGeneratedTopCell{}}(
                /* Fl Fn D Sfa */ Var'Unds'14:SortScheduleCell{},
                /* Fl Fn D Sfa Cl */
                Lbl'-LT-'schedule'-GT-'{}(
                    /* Fl Fn D Sfa Cl */ LblBERLIN'Unds'EVM{}()
                )
            )
        ),
        /* Spa */
        \equals{SortGeneratedCounterCell{}, SortGeneratedTopCell{}}(
            /* Fl Fn D Sfa */ Var'Unds'DotVar0:SortGeneratedCounterCell{},
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'generatedCounter'-GT-'{}(
                /* Fl Fn D Sfa Cl */ \dv{SortInt{}}("0")
            )
        )
    ),
    /* Spa */
    \equals{SortNetworkCell{}, SortGeneratedTopCell{}}(
        /* Fl Fn D Sfa */ Var'Unds'DotVar2:SortNetworkCell{},
        /* Fl Fn D Sfa Cl */
        Lbl'-LT-'network'-GT-'{}(
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'chainID'-GT-'{}(/* Fl Fn D Sfa Cl */ \dv{SortInt{}}("1")),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'activeAccounts'-GT-'{}(
                /* Fl Fn D Sfa Cl */ /* InternalSet: */ Lbl'Stop'Set{}()
            ),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'accounts'-GT-'{}(
                /* Fl Fn D Sfa Cl */
                /* InternalMap: */ Lbl'Stop'AccountCellMap{}()
            ),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'txOrder'-GT-'{}(
                /* Fl Fn D Sfa Cl */ /* InternalList: */ Lbl'Stop'List{}()
            ),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'txPending'-GT-'{}(
                /* Fl Fn D Sfa Cl */ /* InternalList: */ Lbl'Stop'List{}()
            ),
            /* Fl Fn D Sfa Cl */
            Lbl'-LT-'messages'-GT-'{}(
                /* Fl Fn D Sfa Cl */
                /* InternalMap: */ Lbl'Stop'MessageCellMap{}()
            )
        )
    )
)